go.mod

```
@@ -1,8 +1,7 @@
 module github.com/codeready-toolchain/registration-service
 
 go 1.22.0
-
-toolchain go1.22.12
+toolchain go1.24.1
```

this looks wrong

---

let's try what happens if we revert it back to the original version:
```suggestion
toolchain go1.22.12
```